### ISSUE TITLE
Bubbles dissapear or are not rendered on BubbleChart 

### ISSUE THREAD

**Issue by alexhuiculescu** [PhilJay/MPAndroidChart#4179]
**Summary**

On some data sets, the Bubbles are not being rendered, or are rendered strangely. When moving them arround, they dissapear
**Expected Behavior**

The render should be correct with no strange behaviour
**Possible Solution**

**Device (please complete the following information):**
 - Device: Huawei P9Lite
 - Android Version 7.0
 - Library Version (e.g. 3.0.3)

**Additional Context**

This is a gist with the code I used, also with the dataSet.
https://gist.github.com/alexhuiculescu/0e2884f681f622bc093601686e48eb4a

**Comment by YStepiuk** [comment]
Hey!
I've been experiencing kinda same issue. For some reason, my bubbles weren't rendered on the chart.
I don't know what is the issue exactly, but I've found a workaround that works for me.
I've just sorted the BubbleEntries by X in ascending order and it started to work.
Hope it helps

**Comment by alexhuiculescu** [comment]
@YStepiuk yes, this totally works, as the framework is assuming the bubbles are X ordered and displaying them based on that
